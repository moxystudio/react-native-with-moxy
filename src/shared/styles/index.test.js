import * as styles from '.';

it('should export all named styles', () => {
    expect(styles).toMatchSnapshot();
});
