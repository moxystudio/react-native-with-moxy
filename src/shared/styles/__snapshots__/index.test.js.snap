// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should export all named styles 1`] = `
Object {
  "colors": Object {
    "primary": "#221240",
    "secondary": "#2932D3",
    "terciary": "#F64F51",
  },
  "layout": Object {
    "gutter": 20,
  },
  "typography": Object {
    "bold": Object {
      "fontFamily": "Lato-Bold",
      "fontWeight": "normal",
      "paddingBottom": 6,
      "paddingTop": 4,
    },
    "regular": Object {
      "fontFamily": "Lato-Regular",
      "fontWeight": "normal",
      "paddingBottom": 6,
      "paddingTop": 4,
    },
    "thin": Object {
      "fontFamily": "Lato-Thin",
      "fontWeight": "normal",
      "paddingBottom": 6,
      "paddingTop": 4,
    },
  },
}
`;
